src/config.example.ts

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  secretKey: '<Your Secret Here>',
+  mongoUrl: '<Mongo URL>',
+  PORT: 4000,
+  baseURL: 'https://localhost'
```

Comment:
Do we need this parameter? Using it anywhere?

Reply:
Not at the moment, but we might need it while generating callback URLs(if any) in the future.

Reply:
Let's remove it then!